specification/overview.md

```diff
@@ -26,21 +26,52 @@ This specification is designed to be used by three major groups:
+* Don't add dimensions or metrics to the specification just because it can be added.
+* When defining the specification, consideration should be made to existing data already in the major providers' (AWS, GCP, Azure, OCI) data sources.
+* As long as it solves the FinOps use case, there should be a preference to align with data that is already present in a majority of the major providers.
+* Strive for simplicity. However, prioritize accuracy, clarity, and consistency.
+* Strive to build columns that serve a single purpose, with clear and concise names and values.
+* The specification should allow data to be presented free from jargon, using simple understandable terms, and be approachable.
+* Naming and terms used should be carefully considered to avoid using terms for which the definition could be confused by the reader. If a term must be used which has either an unclear or multiple definitions, it should be clarified in the [glossary](#glossary).
+* The specification should provide all of the data elements necessary for the [FinOps capabilities](https://finops.org/framework/capabilities).
 
-### Working Backwards
+### Provider-neutral approach by default
 
-The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the billing data that should be defined in the specification to fulfill that capability. Some of the enabled capabilities will be documented in the [Appendix: FinOps Capabilities Enabled by FOCUS (TODO)](#finopscapabilitiesenabledbyfocus).
+* While the schema, naming, terminology, and attributes of many providers are reviewed during development, this specification aims to be provider-neutral.
+* Contributors must take care to ensure the specification examines how each decision relates to each of the major cloud providers and SaaS vendors, not favoring any single one.
+* In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
 
 ### Extensibility
 
-The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs). The specification however aims to be extensible to SaaS products and other types of cost datasets. Future versions of the specification will look to expand the content to support a broader set of prioritized FinOps capabilities.
+* The initial specification aims to introduce a common schema and terminology for billing datasets produced by Cloud Service Providers (CSPs).
+* The specification, however, aims to be extensible to SaaS products and other types of cost datasets.
```

Comment:
Bit of a chicken and egg problem here. At what point does FOCUS clearly state that SaaS providers should adopt it? Is it at 1.0 since we know it's going to work based on some of the research that was done OR is it some version after 1.0? if it's at 1.0, we should change this line IMO.

Reply:
Without the additional supporting content to assist SaaS vendor adoption ready, stating SaaS should adopt v1.0 seems premature.

Reply:
At the time a release has extended to cover the SaaS use cases it makes sense to update this line.